public/app/features/alerting/unified/components/mute-timings/MuteTimingForm.tsx

```diff
@@ -117,20 +109,7 @@ const MuteTimingForm = ({ muteTiming, showError, loading, provenance }: Props) =
   };
 
   return (
-    <AlertingPageWrapper
```

Comment:
TODO: gotta figure this out, `AlertingPageWrapper` contains the context wrapper but it wants the mute timing information for the page wrapper (this is only fetched after the wrapper is already rendered..)